src/sentry/dynamic_sampling/tasks/recalibrate_orgs.py

```diff
@@ -69,7 +69,11 @@ def recalibrate_orgs(context: TaskContext) -> None:
 )
 def recalibrate_orgs_batch(orgs: Sequence[tuple[int, int, int]]) -> None:
     for org_id, total, indexed in orgs:
-        recalibrate_org(org_id, total, indexed)
+        try:
+            recalibrate_org(org_id, total, indexed)
```

Comment:
Added exception handling to avoid error propagation between orgs in the same batch.